### ISSUE TITLE
WAV is too large. Is there any other type supported?

### ISSUE THREAD

**Issue by lhc-up** [mattdiamond/Recorderjs#195]

**Comment by rescue1155** [comment]
+1

1 minute of audio goes to 5MB, How can we reduce it ?

**Comment by martin-nazaryan** [comment]
Yes, i have the same problem. is there any way to compress audio ?

**Comment by rescue1155** [comment]
You can use the media recorder (The default way) that supported in all major browsers.
For non supported browsers (mostly: safari on iOS devices), you can using media-recorder-polyfill (https://github.com/ai/audio-recorder-polyfill).

That will record the voice in wav format, with best quality 10mints of audio upto 4.5 MBs. (default media recorder).
If polyfill media recorder library will use, That will record about 55% less than that library.

**Comment by xiangyuecn** [comment]
Maybe `MediaRecorder` can be used to convert recorded wav audio into webm. 

Or use lamejs to convert PCM to MP3.

wav blob can get pcm through `wavBlob.slice(44,wavBlob.size,"audio/pcm")`.

demo: https://xiangyuecn.github.io/Recorder/
